pkg/kubelet/kubelet.go

```diff
@@ -741,6 +741,15 @@ func NewMainKubelet(kubeCfg *kubeletconfiginternal.KubeletConfiguration,
 		return nil, err
 	}
 
+	if utilfeature.DefaultFeatureGate.Enabled(features.DevicePlugins) {
```

Comment:
This was mostly legacy implementation. Agree that this should be created at the creation of the Container Manager Object